USAGE.md

```diff
@@ -19,76 +19,125 @@ Now `make install` only supports Linux Ubuntu, CentOS and Mac OS X. If you use o
 you should follow [INSTALL.md](https://github.com/facebook/rocksdb/blob/master/INSTALL.md) to
 install RocksDB manually first. 
 
-### Running in the standalone mode
+### Running in the local standalone mode
 
-1. Start etcd on the default listening port 2379.
 
-    ```sh
-    etcd 
-    ```
-
-2. Start PD on listening port 1234 and cluster ID is 1.
+1. Start PD.
 
     ```sh
-    pd-server -addr 127.0.0.1:1234 --etcd 127.0.0.1:2379 --cluster-id 1 --root /pd
+    pd-server --cluster-id=1 \
+              --addr=127.0.0.1:1234 \
+              --advertise-addr="127.0.0.1:1234" \
+              --http-addr="127.0.0.1:9090" \
+              --etcd-name="default" \
+              --etcd-data-dir="default.pd" \
+              --etcd-listen-peer-url="http://127.0.0.1:2380" \
+              --etcd-advertise-peer-url="http://127.0.0.1:2380" \
+              --etcd-listen-client-url="http://127.0.0.1:2379" \
+              --etcd-advertise-client-url="http://127.0.0.1:2379" \
+              --etcd-initial-cluster="default=http://127.0.0.1:2380" \
+              --etcd-initial-cluster-state="new"
     ```
+    
 
-    Cluster ID is used to distinguish different TiKV clusters.
-    `/pd` is the root prefix path in etcd. 
+    + `cluster-id`: Unique ID to distinguish different PD clusters. It can't be changed after bootstrapping.  
+    + `addr`: Listening address for client traffic. The default official address is `127.0.0.1:1234`.
+    + `advertise-addr`: Advertise address for external client communication. It must be accessible to the PD node.
```

Comment:
"It must be accessible to the PD node."  
It must be accessible from other components?